### PR TITLE
Bug 1975283: Fix skipping of Multi-AZ Cluster Volumes test on GCP with k8s 1.21

### PR DESCRIPTION
**Pull request by yselkowitz**
https://github.com/openshift/kubernetes/pull/825
https://bugzilla.redhat.com/show_bug.cgi?id=1975283

### CODE REVIEW

**Comment by yselkowitz** [comment]
To confirm that this will actually work
/test e2e-gcp

**Comment by yselkowitz** [comment]
Forgot to make update.
/test all

**Comment by yselkowitz** [comment]
Ultimately, I guess this needs to turn into (or be replaced by) a revendoring of kubernetes in origin (after https://github.com/openshift/kubernetes/pull/825 merges) to fix this properly?  If so, as a non-maintainer, that's a bit out of my scope.

**Comment by marun** [comment]
@

> Ultimately, I guess this needs to turn into (or be replaced by) a revendoring of kubernetes in origin (after [openshift/kubernetes#825](https://github.com/openshift/kubernetes/pull/825) merges) to fix this properly? If so, as a non-maintainer, that's a bit out of my scope.

See the following section in the README regarding the steps to take to revendor o/k: https://github.com/openshift/origin/blob/master/README.md#vendoring-from-openshiftkubernetes

**Comment by yselkowitz** [comment]
This is my first revendoring of kubernetes.  While it pulls in much more than needed for this purpose, that is apparently the expected approach, and does fix the e2e-gcp test which is currently broken.
/assign @deads2k

**Comment by yselkowitz** [comment]
Remaining test failure appears to be an existing issue: https://bugzilla.redhat.com/show_bug.cgi?id=1975865

File: test/extended/util/annotate/generated/zz_generated.annotations.go [hunk 1]
```diff
@@ -12035,7 +12035,7 @@ var annotations = map[string]string{
 
 	"[Top Level] [sig-storage] Mounted volume expand Should verify mounted devices can be resized": "Should verify mounted devices can be resized [Suite:openshift/conformance/parallel] [Suite:k8s]",
 
-	"[Top Level] [sig-storage] Multi-AZ Cluster Volumes should only be allowed to provision PDs in zones where nodes exist": "should only be allowed to provision PDs in zones where nodes exist [Suite:openshift/conformance/parallel] [Suite:k8s]",
+	"[Top Level] [sig-storage] Multi-AZ Cluster Volumes should only be allowed to provision PDs in zones where nodes exist": "should only be allowed to provision PDs in zones where nodes exist [Skipped:gce] [Suite:openshift/conformance/parallel] [Suite:k8s]",
```
Comment (yselkowitz) [inline test/extended/util/annotate/generated/zz_generated.annotations.go:12038]: This line is the key part in this PR.

**Comment by danwinship** [comment]
FTR, e2e-metal-ipi-ovn-ipv6 is failing because of things which will be fixed by #26266 except that that PR can't merge because it's failing on e2e-gcp because of the Multi-AZ Cluster Volumes test.

So, like, someone should `/override ci/prow/e2e-metal-ipi-ovn-ipv6`

**Comment by yselkowitz** [comment]
Jenkins is unrelated and being worked: https://github.com/openshift/origin/pull/26268
e2e-metal-ipi-ovn-ipv6 is unrelated and waiting on this: https://github.com/openshift/origin/pull/26266
At this point we need an override to unblock GCP tests
/assign @bparees

**Pull request merged**